lib/http1.c

```diff
@@ -601,20 +601,16 @@ static size_t flatten_headers_estimate_size(h2o_req_t *req, size_t server_name_a
 static size_t flatten_headers(char *buf, h2o_req_t *req, const char *connection)
```

Comment:
should tweak `flatten_headers_estimate_size` too?

Reply:
done in ed0786d